example/SteeringFile_IDEA_o2_v01.py

```diff
@@ -277,10 +279,10 @@
 SIM.gun.isotrop = False
 
 ## Maximal momentum when using distribution (default = 0.0)
-SIM.gun.momentumMax = 10000.0
+SIM.gun.momentumMax = 10.0*GeV
```

Comment:
We set the energy with SIM.gun.energy = 10.0 * GeV which overwrites momentumMax and momentumMin, why do we need to change them?